packages/frontend/src/components/MkEmojiPicker.vue

```diff
@@ -144,6 +153,44 @@ const searchResultCustom = ref<Misskey.entities.CustomEmoji[]>([]);
 const searchResultUnicode = ref<UnicodeEmojiDef[]>([]);
 const tab = ref<'index' | 'custom' | 'unicode' | 'tags'>('index');
 
+const customEmojiFolderRoot: CustomEmojiFolderTree = { value: "", category: "", children: [] };
+
+function parseAndMergeCategories(input: string, root: CustomEmojiFolderTree): CustomEmojiFolderTree {
+  const parts = input.split('/');
+  let category = "";
+  let currentNode: CustomEmojiFolderTree = root;
+
+	for (let part of parts) {
+		if (part) {
+			category += `/${part}`;
+		} else {
+			part = i18n.ts.other
```

Comment:
```suggestion
			part = i18n.ts.other;
```

Lint落ちてるのでここ修正お願いします🙏